### ISSUE TITLE
Weekly Status 2017-10-09

### ISSUE THREAD

**Issue by mrjoro** [ampproject/amphtml#11602]
Please add a comment to this issue with your weekly status update for the AMP Project, such as:

- your accomplishments over the last week
- what you're currently working on
- things you're blocked on
- anything else you think the entire community should know

We encourage updates from everyone who is actively participating in the AMP Project.

**Comment by aghassemi** [comment]
- System Social Share is now disabled in WebView due to Chrome bug see https://github.com/ampproject/amphtml/pull/11556 for details
- UA detection fix for Opera (since Chrome on Android Oreo now has OPR in the UA string!)
- Reducing license text in minified files ( #11589 )

**Comment by cvializ** [comment]
- Presented amp-date-picker design review
- Updating amp-story desktop view with details from ux redlines
- PWA bug report and investigation

**Comment by cramforce** [comment]
- Added `--cache` for making local JS cacheable
- Fixed bug with First Contentful Paint reporting

**Comment by rsimha-amp** [comment]
- Investigated Percy baseline picking logic (https://github.com/percy/percy-client/issues/26, https://github.com/percy/percy-client/issues/27) (master bug #11385)
- More Sauce labs logging fixes (#11512, #11559)
- Integration tests now run on iOS 11! (#11582)
- Working on enabling prod config by default for local testing (#8008, #11588)

**Comment by jridgewell** [comment]
- AMP Layers

**Comment by zhouyx** [comment]
- Met the A4A team in Cambridge
- Grace Hopper Conference : )

**Comment by choumx** [comment]
- Wrote amp-iframe messaging design doc (will be published later)
- Help get `amp-story` merged into amphtml
